modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running multiple applications side-by-side
+* Clear boundaries for operations on what a Mule application is
+* Mule can monitor your applications and will reload configuration changes automatically
+* Applications can depend on different library versions, even if they would conflict before
+* Multiple versions of an application can exist side by side
+
+Every application is deployed together within a *domain*. By default, your application references the default domain, and no further configuration is required. You can see that it gets deployed together with the app in the console:
+
+image::deploy-domain.png[deploy+domain]
+
+If you're deploying multiple applications to the same place, and those applications could share the same resources, then you can create a common *domain* where you can define common configurations that can then be referenced by multiple projects. This allows you to, for example, expose different services in different projects through the same HTTP host and port and be able to deploy everything without any conflicts. xref:shared-resources.adoc[Read More].
```

Comment:
I am not sure why you are rewriting the existing sentence. The bullet makes sense to me,